pom.xml

```diff
@@ -111,7 +111,7 @@
     <pgsql.version>42.6.0</pgsql.version>
     <oracle.groupId>com.oracle.database.jdbc</oracle.groupId>
     <oracle.artifactId>ojdbc8</oracle.artifactId>
-    <oracle.version>21.9.0.0</oracle.version>
+    <oracle.version>23.2.0.0</oracle.version>
```

Comment:
Shouldn't be part of this PR I think

Reply:
Yes, I was in the wrong branch. It's in master too.